.github/workflows/push-and-package.yml

```diff
@@ -7,11 +7,13 @@ on:
     - 'v*'
 
 jobs:
-  build:
-    
+  build-page-1:
```

Comment:
Looks good. I guess the jobs can't be dynamically generated?

Reply:
Initially I didn't think so because all the examples for the [matrix](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix) property are about the OS, but I took a look again just now and it appears I can add any variable to the mix.  I'm testing a variant that uses it and will move that here assuming it works.

Reply:
Yep!  It worked.  🎉